src/main/java/seedu/address/logic/commands/AddPlanCommand.java

```diff
@@ -85,8 +85,11 @@ public boolean equals(Object other) {
         if (!(other instanceof AddPlanCommand)) {
             return false;
         }
+
         AddPlanCommand otherAddPlanCommand = (AddPlanCommand) other;
-        return planName.equals(otherAddPlanCommand.planName)
+        String thisPlan = planName.toString().toLowerCase();
+        String otherPlan = otherAddPlanCommand.planName.toString().toLowerCase();
```

Comment:
Hi Jannice, just a suggestion: Do you think it would be more OOP-friendly to have a .getLowerCase method in the PlanName class?

Doing so would mean that we have to do the same thing with add friends though

Reply:
I think that's a great idea, let me fix it